### PR TITLE
Spec

### PR DESCRIPTION
**Pull request by skrzepto**
adding spec file for fedora app for this project.

for some reason when python setup.py test is run it tries to download faker which isn't listed anywhere in the files so its commented out for now.

### CODE REVIEW

**Comment by skrzepto** [comment]
having difficulties with using github archive for `Source0`

**Comment by skrzepto** [comment]
spec file passes koji build just need to figure out what to do about `Source0`

File: python-fmn-sse.spec [hunk 1]
```diff
@@ -1,55 +1,57 @@
...
 Name:           python-%{pkgname}
-Version:        0.1
+Version:        0.1.0
 Release:        1%{?dist}
 Summary:        %{sum}
 
 License:        GPL
 URL:            https://github.com/fedora-infra/%{modname}
-Source0:        https://github.com/fedora-infra/%{modname}/archive/develop.zip
+Source0:        https://github.com/fedora-infra/fmn.sse/archive/v%{version}.zip
```
Comment (pypingou) [inline python-fmn-sse.spec]: you could have left `%{modname}` if you wanted

File: python-fmn-sse.spec [hunk 2]
```diff
@@ -1,55 +1,57 @@
...
 %prep
-%autosetup -n %{pkgname}-%{version}
+%autosetup -n %{modname}-%{version}
 
 # Remove bundled egg-info in case it exists
 rm -rf %{pkgname}.egg-info
 
 %build
-%py2_build
+%{__python2} setup.py build
```
Comment (pypingou) [inline python-fmn-sse.spec]: I think the former was good

File: python-fmn-sse.spec [hunk 3]
```diff
@@ -1,55 +1,57 @@
...
 
 %install
-%py2_install
+%{__python2} setup.py install -O1 --skip-build --root=%{buildroot}
```
Comment (pypingou) [inline python-fmn-sse.spec]: same here

**Comment by pypingou** [comment]
Couple of comment but nothing blocking :+1:

**Comment by skrzepto** [comment]
@pypingou my concern for `Source0` was that the link is `https://github.com/fedora-infra/fmn.sse/archive/v%{version}.zip` so it's looking for a file called `v0.1.0.zip` but the actuall file downloaded is `fmn.sse-v0.1.0.zip  `

any thoughts to fix that? I had to manually change the zip file name to get the build to work

**Comment by pypingou** [comment]
For Source0 this page may help https://fedoraproject.org/wiki/Packaging:SourceURL#Git_Hosting_Services

**Comment by pypingou** [comment]
So something like https://github.com/fedora-infra/fmn.sse/archive/v0.1.0.zip#fmn-sse-v0.1.0.zip may work?

**Comment by pypingou** [comment]
Worst case the easiest is to upload the tarball to pypi and use that URL in the spec file :)

**Comment by skrzepto** [comment]
pypingou just missing `/` after the `#` and it works awesome :) thanks!

**Pull request merged**